app/code/Magento/CustomerGraphQl/Model/Customer/GetAllowedCustomerAttributes.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\CustomerGraphQl\Model\Customer;
+
+use Magento\Customer\Api\CustomerMetadataManagementInterface;
+use Magento\Customer\Api\Data\CustomerInterface;
+use Magento\Customer\Api\Data\CustomerInterfaceFactory;
+use Magento\Eav\Model\AttributeRepository;
+use Magento\Eav\Model\Entity\Attribute\AbstractAttribute;
+use Magento\Framework\Api\SearchCriteriaBuilder;
+use Magento\Framework\Exception\InputException;
+use Magento\Framework\GraphQl\Exception\GraphQlInputException;
+use Magento\Framework\Reflection\DataObjectProcessor;
+
+/**
+ * Get allowed address attributes
+ */
+class GetAllowedCustomerAttributes
```

Comment:
Can we reuse `\Magento\Customer\Model\Metadata\Form::getAllowedAttributes`?

Reply:
Also, need to take into account `scope` of attribute

Reply:
@naydav 
We will have to use form code and pull a lot of dependencies and may end up with the inability to do partial data update. 
Thanks!